### DISCUSSION TITLE
播放歌曲无法暂停

### DISCUSSION THREAD

**Discussion by yzh990918** [qier222/YesPlayMusic#1191] [Q&A]
# 尽量每个 issue 只提一个 bug 或新功能

### 提新 issue 前请确认 👉

- 没人提过这个 issue（[这里看所有 issue](https://github.com/qier222/YesPlayMusic/issues)）
- 项目的 Todo 里没有与你 issue 相关的内容（[这里看 Todo](https://github.com/qier222/YesPlayMusic/projects/1)）

### 反馈 bug 需要的信息

- 用的是网页版还是客户端
- 浏览器名称或电脑操作系统
- 控制台 Console 页面的截图（按 F12 可打开控制台）

![image](https://user-images.githubusercontent.com/46418596/148355676-7507a835-4d6b-4003-85b2-21d57e004243.png)

**Comment by pan93412** [comment]
請重設 App。

要重設 App，請在 Console 輸入 `resetApp()` 然後 Enter。
